src/cluster/kv/util/util.go

```diff
@@ -279,42 +281,42 @@ func updateWithKV(
 	return nil
 }
 
-func logNilUpdate(logger log.Logger, k string, v interface{}) {
-	getLogger(logger).WithFields(
-		log.NewField("key", k),
-		log.NewField("default-value", v),
+func logNilUpdate(logger *zap.Logger, k string, v interface{}) {
+	getLogger(logger).With(
+		zap.String("key", k),
+		zap.Any("default-value", v),
```

Comment:
Some log fields are kebab-case and others are camelCase. Can we stick to one convention?

Reply:
Fair, original code was across two repos/authors. I'll standardise to camel.

Reply:
done